css/style.css

```diff
@@ -0,0 +1,27 @@
+#main{
+  text-align: center;
+  margin: 30px 8px;
+    padding: 15px;
+    border-radius: 5px;
+    background: #eee;
+}
+#image {
+  max-width: 100%;
+  height: auto;
+  display: block;
+  margin: 0 auto;
+}
+#img-div{
+  background: white;
```

Comment:
### [Optional]
- [ ] Please use some emojis to make your review human-friendly:blush: